File: jmx-monitor-web/src/main/app-web/src/app/jmx-connection/model/jmx.connection.model.ts
export class JmxConnectionModel {
  /**
   * 连接方式
   */
  mode: string;

  /**
   * ip 地址
   */
  ipAddress: string;

  /**
   * 连接端口
   */
  port: string;

  /**
   * 连接 url
   */
  url: string;

  constructor (mode, ipAddress, port) {
    this.mode = mode;
    this.ipAddress = ipAddress;
    this.port = port;
  }

}

File: jmx-monitor-web/src/main/java/com/ztesoft/jmx/monitor/model/JmxConnectionModel.java
package com.ztesoft.jmx.monitor.model;

/**
 * @Author: 陈名立
 * @Description:
 * @Date: 2017, 九月 03 16:03 on 星期日
 */
public class JmxConnectionModel {
    private String mode;
    private String ipAddress;
    private String port;

    public String getMode() {
        return mode;
    }

    public void setMode(String mode) {
        this.mode = mode;
    }

    public String getIpAddress() {
        return ipAddress;
    }

    public void setIpAddress(String ipAddress) {
        this.ipAddress = ipAddress;
    }

    public String getPort() {
        return port;
    }

    public void setPort(String port) {
        this.port = port;
    }

    @Override
    public String toString() {
        return "JmxConnectionModel{" +
                "mode='" + mode + '\'' +
                ", ipAddress='" + ipAddress + '\'' +
                ", port='" + port + '\'' +
                '}';
    }
}
